content/en/real_user_monitoring/browser/setup/server/java_servlet.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Java Servlet
+beta: true
+code_lang: java
+type: multi-code-lang
+code_lang_weight: 7
+further_reading:
+- link: '/real_user_monitoring/browser/setup/server/'
+  tag: 'Documentation'
+  text: 'Browser Monitoring Auto-Instrumentation'
+---
+
+{{< callout url="https://forms.gle/RdVvDsmmzW21s2vd7" header="Join the Preview!">}}
+  RUM Auto-Instrumentation through Java servlet-based web servers is in Preview. 
+{{< /callout >}}
+
+## Overview
+
+RUM Auto-Instrumentation automatically adds RUM monitoring to your web application server. This means you don't need to modify your frontend code directly. Instead, you can start collecting RUM data by editing a configuration file. However, if you want to track specific user actions (custom actions) or add custom event details (event attributes), you still need to add some code to your application.
+
+## How it works
+
+The Java SDK works by monitoring your servlet API (which handles web requests and responses). It automatically checks each HTTP response your server sends. When it detects an HTML response, it looks for the `<head>` section and automatically adds the RUM JavaScript code there. The rest of your HTML content remains unchanged. This approach adds minimal overhead, so it works well even with streaming responses (where content is sent in real-time).
```

Comment:
```suggestion
The Java SDK works by monitoring your servlet API (which handles web requests and responses), automatically checking each HTTP response your server sends. When it detects an HTML response, it looks for the `<head>` section and automatically adds the RUM JavaScript code there. The rest of your HTML content remains unchanged. This approach adds minimal overhead, so it works well even with streaming responses (where content is sent in real-time).
```
When I saw `It automatically checks...`, I wasn't sure if `it` referred to `the Java SDK` or `your servlet API`. I think this might help focus the phrase around the SDK for the full paragraph without repeating it too much, but feel free to edit as needed!